scraps.txt

```diff
@@ -0,0 +1,22 @@
+You are now connected to database "nc_news_test" as user "david".
```

Comment:
Can add this file to your .gitignore, will probably have to remove it from git as well though - don't worry if that'll be a pain, can leave it.